ios/Classes/ComAppceleratorUrlSessionSessionConfigurationProxy.m

```diff
@@ -23,9 +24,20 @@ - (id)_initWithPageContext:(id<TiEvaluator>)context andArguments:(NSDictionary*)
             sessionConfiguration = [NSURLSessionConfiguration backgroundSessionConfiguration:identifier];
         }        
     }
-    
     return self;
 }
+-(void)_initWithProperties:(NSDictionary *)properties
+{
+    
+    NSString *identifier = [properties objectForKey:@"identifier"];
+    
+    if ([TiUtils isIOS8OrGreater] == YES) {
+        sessionConfiguration = [NSURLSessionConfiguration backgroundSessionConfigurationWithIdentifier:identifier];
+    } else {
+        sessionConfiguration = [NSURLSessionConfiguration backgroundSessionConfiguration:identifier];
+    }
+    [super _initWithProperties:properties];
```

Comment:
This whole logic has already been done in `_initWithPageContext:andArguments` above, why would you do it here as well?